src/ripple/app/ledger/impl/MetaView.cpp

```diff
@@ -543,11 +543,12 @@ Json::Value MetaView::getJson (int) const
 
 void
 MetaView::apply (BasicView& to,
-    STTx const& tx, TER ter, beast::Journal j)
+    std::shared_ptr<STTx const> const& tx,
```

Comment:
Ok, I know what happened here. I started doing this when I was thinking that the queue would live inside the `TransactionEngine`. Then we talked about what a bad idea that was, but I didn't undo the `shared_ptr` changes, because there was some mention that `shared_ptr` would be somehow nicer anyway. Then it propagated to `MetaView` to satisfy dependencies.

I can rip them out pretty easily.